### PR TITLE
feat(user creation) display_name validation error

### LINKED ISSUE #3058: Async validation for user display name is unclear
We have an async validator for the display name of a user on the user creation form.  I had forgotten it exists because it is so discrete.  Here it is below:

erroronduplicateusername
_Fig 1: Async validator for display name_

The above indicator shows an error for a duplicate display name - "admin" has already been used.  The following changes need to be made:
 1. Include text that says "this display name is already in use"
 2. Color the input, and the "X" and any accompanying text, in red.

### PR DESCRIPTION
**Pull request by jeremielodi**
closes #3058

### CODE REVIEW

**Comment by jniles** [comment]
Could you include a screenshot?

**Comment by jeremielodi** [comment]
val

File: client/src/modules/users/user.modal.html [hunk 1]
```diff
@@ -21,7 +21,10 @@
       <div>
         <input name="display_name" bh-unique="/users/{{UserModalCtrl.user.display_name}}" ng-model="UserModalCtrl.user.display_name" autocomplete="off" ng-maxlength="UserModalCtrl.maxLength" class="form-control" required>
 
-        <span ng-show="UserForm.display_name.$error.unique" class="glyphicon glyphicon-remove form-control-feedback" id="unique-error-icon"></span>
+        <div ng-show="UserForm.display_name.$error.unique" translate class="text-danger">
```
Comment (jniles) [inline client/src/modules/users/user.modal.html]: From your screenshot, looks like you need to make this a `<span>` and have [class="help-block"](https://getbootstrap.com/docs/3.3/css/#forms-help-text) on it, to make it behave the same as the other help blocks.

**Comment by jniles** [comment]
I've switched up the classes a bit based on [bootstrap's documentation](http://getbootstrap.com/docs/3.3/css/#forms-control-validation).  It looks like the above.

**Pull request merged**